indico/modules/events/api.py

```diff
@@ -341,6 +344,19 @@ def _build_session_api_data(self, session_):
         return [self._serialize_session_block(b, serialized_session, session_access_list, can_manage)
                 for b in session_.blocks]
 
+    @staticmethod
+    def serialize_reference(reference):
+        """Return the data for a reference"""
+        return {
+            'type': reference.reference_type.name,
+            'value': reference.value
+        }
+
+    @classmethod
+    def serialize_references(cls, references):
+        """Return the data for a list of references"""
+        return map(cls.serialize_reference, references)
```

Comment:
No strong opinion either way, I'd probably maybe keep `serialize_reference` but move this `map()` in the place where `serialize_references` is called atm.